Formula/aws-sso-util.rb

```diff
@@ -19,22 +19,22 @@ class AwsSsoUtil < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "aef2aaa71e70de99b18828a65103208d1c0449db3f12fcacd28166c03674d5cc"
   end
 
-  depends_on "rust" => :build
   depends_on "python@3.11"
   depends_on "six"
 
-  on_linux do
-    depends_on "pkg-config" => :build
-  end
-
   resource "attrs" do
-    url "https://files.pythonhosted.org/packages/1a/cb/c4ffeb41e7137b23755a45e1bfec9cbb76ecf51874c6f1d113984ecaa32c/attrs-22.1.0.tar.gz"
-    sha256 "29adc2665447e5191d0e7c568fde78b21f9672d344281d0c6e1ab085429b22b6"
+    url "https://files.pythonhosted.org/packages/21/31/3f468da74c7de4fcf9b25591e682856389b3400b4b62f201e65f15ea3e07/attrs-22.2.0.tar.gz"
+    sha256 "c9227bfc2f01993c03f68db37d1d15c9690188323c067c641f1a35ca58185f99"
   end
 
   resource "aws-error-utils" do
     url "https://files.pythonhosted.org/packages/4e/7b/622c18e41b17935ac72f4f7b8775e18fe6dd6ecca0d1068fd95f5cbd91f9/aws-error-utils-1.3.0.tar.gz"
     sha256 "188159a8897552408dc3545aed55b49a12532cbde841aad0490e2b93a1275cfc"
+
+    # Use poetry_core backend to avoid unnecessary cmake and rust build dependencies.
+    # Remove when release uses aws-error-utils>=2.6.
+    # Backport of https://github.com/benkehoe/aws-error-utils/commit/f23a4bb8703e8ffabf1d008e04495572cc12d4ee
+    patch :DATA
```

Comment:
One demerit is patch may break `brew bump` logic.

Other options are:
- Add `cmake` (and keep `rust`) dependency. Demerit is these are probably unnecessary.
- Use `poetry` as dependency and manually run build command. Demerit is that we cannot use `virtualenv_install_with_resources` DSL which makes it harder to maintain.

Reply:
link to [that PR](https://github.com/benkehoe/aws-error-utils/pull/6)

Reply:
maybe we should also file a PR to update `aws-sso-uti` to use `aws-error-utils>=2.6`